book/src/running-validator/validator-info.md

```diff
@@ -4,7 +4,7 @@ You can publish your validator information to the chain to be publicly visible t
 
 ## Run solana validator-info
 
-Run the solana CLI to populate a validator-info account:
+Run the solana CLI to populate a validator info account:
```

Comment:
This change niggles at me, although I won't bother to change it back. Until such time as we generalize "info accounts", "validator-info" is the type of account.